catalog/resource_quality_monitor.go

```diff
@@ -53,7 +53,6 @@ func ResourceQualityMonitor() common.Resource {
 			common.CustomizeSchemaPath(m, "profile_metrics_table_name").SetReadOnly()
 			common.CustomizeSchemaPath(m, "status").SetReadOnly()
 			common.CustomizeSchemaPath(m, "dashboard_id").SetReadOnly()
-			common.CustomizeSchemaPath(m, "schedule", "pause_status").SetReadOnly()
```

Comment:
Hi @xinyijiang-db , thanks for the PR, this doesn't remove the pause_status from schema, only removes it not being set to read only. Also, isn't this a breaking change? 

If this is needed to be done, then it would also need to be done here: https://github.com/databricks/terraform-provider-databricks/blob/main/internal/providers/pluginfw/resources/qualitymonitor/resource_quality_monitor.go.

Reply:
> @alexott let me clarify a little bit more. we still want it in the response so that users can see it but we don't want it in the update function request to our service

> it seems that terraform does a read before it calls our update function, which causes our service to reject the update request

@xinyijiang-db yeah if you want to remove it from schema, you need to delete it explicitly, not just remove the read only status. Alternatively, in the `Update` section of the resource, you could also empty the field there so it's not sent in the request